test/FunctionalTests/Microsoft.OData.Core.Tests/ODataMessageWriterTests.cs

```diff
@@ -999,10 +1003,54 @@ public async void DisposeAsync_Should_Dispose_Stream_Asynchronously_AfterWriting
 
             Assert.True(stream.Disposed);
         }
+
+        [Fact]
+        public void WriteMetadataDocument_WorksForLegacyScaleAndSridVariables()
+        {
+            // Arrange
+            IEdmModel edmModel = GetScaleAndSridModel();
+
+            // Act
+            string payload = this.WriteAndGetPayloadForScaleAndSridVariables(edmModel, useLegacyScaleAndSridVariable : true, omWriter =>

```

Comment:
```suggestion
            string payload = this.WriteAndGetPayloadForScaleAndSridVariables(edmModel, useLegacyScaleAndSridVariable : true, messageWriter =>
```
Because `messageWriter` is obviously a better variable name than `omWriter`